src/coreclr/nativeaot/Runtime/amd64/GcProbe.asm

```diff
@@ -36,7 +38,7 @@ PUSH_PROBE_FRAME macro threadReg, trashReg, BITMASK
     lea             trashReg, [rsp + 0]         ; trashReg == address of frame
 
     ;; allocate scratch space and any required alignment
-    alloc_stack     20h + 10h
+    alloc_stack     20h + 8h
```

Comment:
```suggestion
    alloc_stack     20h + 18h
```
I think this needs enough space for xmm0 saved on the next line to fit

Reply:
Thanks! I didn't fully understand what it does but I _think_ I know now.